packages/nextra/src/plugin.ts

```diff
@@ -183,15 +179,7 @@ export async function collectFiles(
 
     const metaPath = path.join(dir, metaFilename) as MetaJsonPath
 
-    if (metaIndex === undefined) {
-      // Create a new meta file if it doesn't exist.
-      fileMap[metaPath] = {
-        kind: 'Meta',
-        ...(locale && { locale }),
-        data: Object.fromEntries(defaultMeta)
-      }
-      items.push(fileMap[metaPath])
-    } else {
```

Comment:
@B2o5T It seems like this change caused `callout` and `tabs` to be lowercased:

<img width="236" alt="CleanShot 2023-01-24 at 14 23 13@2x" src="https://user-images.githubusercontent.com/3676859/214305878-e2323bbc-4751-44d5-9b9f-fcc8852a2631.png">

https://nextra.site/docs/docs-theme/built-ins

Reply:
Saw it yesterday, will fix it !